xarray/core/dask_array_ops.py

```diff
@@ -24,3 +26,70 @@ def dask_rolling_wrapper(moving_func, a, window, min_count=None, axis=-1):
     # trim array
     result = da.ghost.trim_internal(out, depth)
     return result
+
+
+def rolling_window(a, axis, window, center, fill_value):
+    """ Dask's equivalence to np.utils.rolling_window """
+    orig_shape = a.shape
+    # inputs for ghost
+    if axis < 0:
+        axis = a.ndim + axis
+    depth = {d: 0 for d in range(a.ndim)}
+    depth[axis] = int(window / 2)
+
+    offset = 1 if window % 2 == 0 else 0
+
+    # pad the original array before the operation in order to avoid copying
+    # the output array (output array is just a view).
+    if center:
+        start = int(window / 2)  # 10 -> 5,  9 -> 4
+        end = window - 1 - start
+    else:
+        start, end = window - 1, 0
+
+    drop_size = depth[axis] - offset - np.maximum(start, end)
```

Comment:
Normally I think of `size` as a positive integer, but below you use `-drop_size` to make it positive. I think this would be clearer as `drop_size = max(start, end) - offset - depth[axis]` (use `max()` vs `np.maximum` as `start` and `end` are Python integers)

Reply:
You are right. I thought it becomes sometimes negative.
Fixed.